released/packages/coq-coquelicot/coq-coquelicot.3.0.3+8.11/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "guillaume.melquiond@inria.fr"
+homepage: "http://coquelicot.saclay.inria.fr/"
+dev-repo: "git+https://gitlab.inria.fr/coquelicot/coquelicot.git"
+bug-reports: "https://gitlab.inria.fr/coquelicot/coquelicot/issues"
+license: "LGPL 3-only"
+build: [
+  ["./autogen.sh"]
+  ["./configure"]
+  ["./remake" "-j%{jobs}%"]
+]
+install: ["./remake" "install"]
+depends: [
+  "coq" {= "8.11.0"}
+  "coq-mathcomp-ssreflect" {>= "1.6"}
+  ("conf-g++" {build} | "conf-clang" {build})
+  "conf-autoconf" {build}
+]
+tags: [ "keyword:real analysis" "keyword:topology" "keyword:filters" "keyword:metric spaces" "category:Mathematics/Real Calculus and Topology" "date:2020-01-14" "logpath:Coquelicot"]
+authors: [ "Sylvie Boldo <sylvie.boldo@inria.fr>" "Catherine Lelay <catherine.lelay@inria.fr>" "Guillaume Melquiond <guillaume.melquiond@inria.fr>" ]
+synopsis: "(patched for Coq 8.11 compatibility by MSoegtropIMC) A Coq formalization of real analysis compatible with the standard library"
```

Comment:
```suggestion
synopsis: "A Coq formalization of real analysis compatible with the standard library (patched for Coq 8.11 compatibility by MSoegtropIMC)"
```
so as not to obscure what the library does.

Reply:
I put it at the begin because in some forms of opam lists the end of the synopsis is not visible and I don't want anybody to be mistaken about this. It would be easier if we could agree on an official shorthand notation for "coq platform patched versions". I would still put it at the begin then.